app/scripts/controllers/swaps.js

```diff
@@ -91,7 +91,8 @@ export default class SwapsController {
 
     this.indexOfNewestCallInFlight = 0
 
-    this.ethersProvider = new ethers.providers.Web3Provider(provider)
+    // The network ID is hard-coded as "1" because swaps is only used on Mainnet
+    this.ethersProvider = new ethers.providers.Web3Provider(provider, 1)
```

Comment:
We should maybe `parseInt` a network controller constant here for searchability/clarity for uninformed readers.

Reply:
This has been done